storage/src/mmr/verification.rs

```diff
@@ -197,7 +198,7 @@ impl<H: CHasher> Proof<H> {
                     }
                 }
             } else {
-                hashes.push(mmr.get_node(item.0).unwrap().clone());
+                hashes.push(mmr.get_node(item.0).await.unwrap().clone());
```

Comment:
seems like we shouldn't panic here (if a node doesn't exist)?

Reply:
Yeah certainly not on runtime / storage error, will want to propagate that up.  But for the case where the element was pruned, the implementation assumes the caller has checked that all necessary items are present (see function doc).  Could still propagate this up anyway I suppose?

Reply:
changed to propagate the runtime error (missing node still panics)